media/doc/For_Turkish_Translation.txt

```diff
@@ -1,7 +1,3 @@
-The ReactOS is translating to pure Turkish with new terminology. For new terminology, see: http://eersoy93.blogspot.com.tr/p/sozluk.html (in Turkish)
+ReactOS isn't translated to pure Turkish anymore by me but I may translate ReactOS to normal Turkish. For pure Turkish words, prefixes, suffixes etc. see for my pure Turkish dictionary: https://github.com/eersoy93/sozluklerim/blob/master/%C3%96z_T%C3%BCrk%C3%A7e_S%C3%B6zl%C3%BCk.md
 
```

Comment:
This change seems to go beyond updating your email address, and I do not understand it.

Reply:
My commit has group of changes by me and my original commit's header is: Make some changes

Reply:
@AmineKhaldi Should I separate the commit to two commits?

Reply:
We generaly don't like huge comits that span the whole codebase.